src/Compilers/Core/Portable/Operations/IBranchOperationExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    public static class IBranchOperationExtensions
+    {
+        /// <summary>
+        /// Gets a loop operation that corresponds to the given branch operation.
+        /// </summary>
+        /// <param name="branchOperation">the branch operation for which a corresponding loop is looked up</param>
```

Comment:
>t [](start = 42, length = 1)

Capitalization

Reply:
And elsewhere in the extensions.

---
In reply to: [200513121](https://github.com/dotnet/roslyn/pull/28285#discussion_r200513121) [](ancestors = 200513121)

Reply:
I can bet I had found an example somewhere around and just followed it. But np, I'll correct.